### PR TITLE
Cherry pick docs fixes

### PR DESCRIPTION
**Pull request by SvenDowideit**

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@nathanleclaire @londoncalling

File: docs/completion.md [hunk 1]
```diff
@@ -31,18 +31,13 @@ Place the completion scripts in `/etc/bash_completion.d/` (`` `brew --prefix`/et
...
+### Zsh
 
 Place the completion scripts in your `/path/to/zsh/completion`, using e.g. `~/.zsh/completion/`
 
     mkdir -p ~/.zsh/completion
-    files=(docker-machine docker-machine-wrapper docker-machine-prompt)
-    for f in "${files[@]}"; do
-      curl -L https://raw.githubusercontent.com/docker/machine/v$(docker-machine --version | tr -ds ',' ' ' | awk 'NR==1{print $(3)}')/contrib/completion/zsh/$f > ~/.zsh/completion/_$f
-    done
+    curl -L https://raw.githubusercontent.com/docker/machine/v$(docker-machine --version | tr -ds ',' ' ' | awk 'NR==1{print $(3)}')/contrib/completion/zsh/_docker-machine > ~/.zsh/completion/_docker-machine
```
Comment (thaJeztah) [inline docs/completion.md]: these aren't in the current release yet https://github.com/docker/machine/tree/v0.7.0/contrib/completion so probably needs to be removed from the cherry picks

**Comment by thaJeztah** [comment]
one note, but I think the others should be safe to go

**Comment by SvenDowideit** [comment]
@londoncalling and I decided to wait til @nathanleclaire is back :)

**Comment by thaJeztah** [comment]
SGTM

**Comment by nathanleclaire** [comment]
@SvenDowideit Let me know if this one is good to go.

**Comment by londoncalling** [comment]
@nathanleclaire I believe we were just waiting for your ok to make sure that cherry-picking these wasn't referring to anything that isn't applicable in current release of Docker Machine. From what I can tell, all is good since one of the changes (`docs/install-machine.md`) is an update to the `curl` download command to reference v. 0.7.0 of machine, which is the current version.

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**